packages/markdown-docx/src/CiceroMarkToOOXML/rules.js

```diff
@@ -78,4 +78,39 @@ const HEADING_RULE = (value, level) => {
     `;
 };
 
-module.exports = { TEXT_RULE, EMPHASIS_RULE, HEADING_RULE };
+/**
+ * Inserts a variable.
+ *
+ * @param {string} title Title of the variable. Eg. receiver-1, shipper-1
+ * @param {string} tag   Name of the variable. Eg. receiver, shipper
+ * @param {string} value Value of the variable
+ * @param {string} type  Type of the variable - Long, Double, etc.
+ * @returns {string} OOXML string for the variable
+ */
+const VARIABLE_RULE = (title, tag, value, type) => {
+    return `
+      <w:sdt>
+        <w:sdtPr>
+          <w:rPr>
+            <w:color w:val="000000"/>
+            <w:sz w:val="24"/>
+            <w:highlight w:val="green"/>
+          </w:rPr>
+          <w:alias w:val="${titleGenerator(title, type)}"/>
+          <w:tag w:val="${tag}"/>
+        </w:sdtPr>
+        <w:sdtContent>
+          <w:r>
+            <w:rPr>
+              <w:color w:val="000000"/>
```

Comment:
I am not sure if this would be necessary as the default colour is black only. Could you confirm?

Reply:
@algomaster99 Yes the default color is black. This can be removed.

Reply:
@algomaster99 I will create a PR in the morning.

Reply:
Sure. Don't create a new one. You can update this one only.